src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/DungeonTimer.kt

```diff
@@ -318,7 +318,7 @@ class DungeonTimer {
         override fun render() {
             val player = mc.thePlayer
             val world: World? = mc.theWorld
-            if (toggled && Utils.inDungeons && player != null && world != null && bossEntryTime != -1L && DungeonFeatures.dungeonFloor == "F6" || DungeonFeatures.dungeonFloor == "M6") {
```

Comment:
You can try to use parenthesis here or use Utils.equalsOneOf instead of repeating the same constraints.